scss/config/_global-settings.scss

```diff
@@ -12,9 +12,102 @@ $decanter-default-settings: (
   breakpoint-helper-selector: #breakpoint-helper,
 );
 
-/// @name Bourbon Settings
-///
-/// @group settings
-$bourbon: (
-  "modular-scale-ratio": 1.333,
+$neat-grid: (
+    columns: $grid-columns,
+    gutter:  $gutter,
+    media: null,
+    color: rgba(#00d4ff, 0.25),
+    direction: ltr,
+);
+
+// Lower boundaries
+// ----------------
+
+/// Screen XS Min
+$screen-xs-min:               0;
+
+/// Screen SM Min
+$screen-sm-min:               576px;
+
+/// Screen MD Min
+$screen-md-min:               768px;
+
+/// Header MD Min
+$header-md-min:               860px;
```

Comment:
This is the only new breakpoint and should be the only variable you need to declare.